parse.go

```diff
@@ -661,7 +674,7 @@ func (p *Parser) process(args []string) error {
 		// lookup the spec for this option (note that the "specs" slice changes as
 		// we expand subcommands so it is better not to use a map)
 		spec := findOption(specs, opt)
-		if spec == nil {
+		if spec == nil || opt == "" {
```

Comment:
given the change above re specifying "no short option" by omitting a short option from the tag, is it possible to drop this change?

Reply:
👍🏻